exercises/grade-school/.meta/hints.md

```diff
@@ -1,3 +1,4 @@
 ## Hints
 For this exercise the following F# feature comes in handy:
 - The [Map](https://en.wikibooks.org/wiki/F_Sharp_Programming/Sets_and_Maps#Maps) type associates keys with values. It is very similar to .NET's `Dictionary<TKey, TValue>` type, but with one major difference: `Map` is [immutable](https://fsharpforfunandprofit.com/posts/correctness-immutability/).
+- A [type abbreviation (or alias)](https://fsharpforfunandprofit.com/posts/type-abbreviations/) to a `Map` type used as a `School` type. 
```

Comment:
I would slightly refrase this, to not have it tied to the actual exercise's details. What about:

"A type abbreviation is used to create a descriptive alias for a more complex type"